### ISSUE TITLE
Filter ignored issues out of weekly report

### ISSUE THREAD

**Issue by ataylormays-duetto** [getsentry/sentry#52540]
### Problem Statement

The Weekly Report from Sentry includes a very helpful section called "Issues with the most errors" that shows the top 3 issues from the previous week with the highest event count. However, that list includes issues which have been ignored, which are less important to our org than those which have not been ignored.

### Solution Brainstorm

A couple possible solutions:

- Allow configuration for the weekly report to allow the report to filter out issues w/ certain statuses. It would also be helpful to configure the number of issues in the "Issues with the most errors" section
- Add a link in the report that when clicked on takes the user to a view of the top issues, which would allow us to easily see >3 issues

### Product Area

Issues

**Comment by ceorourke** [comment]
cc @rachrwang honestly not sure if it's intended to include ignored issues in here - thanks for the issue @ataylormays-duetto

**Comment by alowdon** [comment]
Is there any further progress on this? We would also like to be able to remove archived issues from this summary; if we have archived issues it is because we don't think they need further attention (at least at the moment), and so having them included in this email is a distraction, especially since there are only three issues shown in this space.

We have ended up deleting and discarding future events for some errors as a workaround, but that is a bit aggressive and it would be nicer to be able to have the issues still recorded but archived instead.
